ibis/__init__.py

```diff
@@ -1,12 +1,14 @@
 """Initialize Ibis module."""
+import warnings
+
 import pkg_resources
 
+# Converting an Ibis schema to a pandas DataFrame requires registering
```

Comment:
Sorry, do you mean `converting an Ibis schema to a pandas dtype` here? I am not sure what do you mean by converting ibis schema to a pandas DataFrame..

Reply:
Also IMO, I think it's OK to treat pandas backend special since pandas can be a hard dependency and is kind of baked in the ibis core/expr layer.

Reply:
Thanks, yes, this is not clear, will rephrase. This refers to the `Schema.apply_to` I explained below.

Personally I don't think pandas should be a hard dependency, and the pandas backend shouldn't be preloaded. I don't even think the pandas backend should be shipped with Ibis in the long term. You can see my proposed roadmap at #2828, so you can have an idea on my thoughts for the project. And any feedback there is very welcome.

Reply:
but please do not change at this time. pandas is not a big deal as a dep at all.

Reply:
Mhh, the change is not to make pandas not be a dependency (even if it would help if we do that). The change is trivial, just changing the function to the obvious file where it should be. And also makes it a regular method, instead of injecting it as a method in a hacky way. I think it's a small, but useful clean up, with no drawbacks. What is the problem with keeping it?